packages/braintree-utils/src/utils/is-braintree-connect-profile-data.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { getBraintreeConnectProfileDataMock } from '../';
+
+import isBraintreeConnectProfileData from './is-braintree-connect-profile-data';
+
+describe('isBraintreeConnectProfileData()', () => {
+    it('returns true if profile data belong to connect', () => {
```

Comment:
May I ask you to cover test where isBraintreeConnectProfileData returns false?

Reply:
Added